.gitignore

```diff
@@ -7,6 +7,7 @@ Makefile
 makefile
 build*
 prebuilt
+cmake-build-debug/
```

Comment:
I have never seen this directory if it's specific to your setup then it probably shouldn't be in our .gitignore. Just call your build directory 'build' and it will be ignored.